src/mmw/js/src/analyze/models.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 var $ = require('jquery'),
-    lodash = require('lodash'),
+    _ = require('lodash'),
```

Comment:
😑  ... we should just kick underscore and use lodash exclusively, and since both aren't being imported in this file there shouldn't be any confusion ...

... but yeah, this change is not necessary for this work. Will remove and use `lodash`.